v2/utils.go

```diff
@@ -0,0 +1,35 @@
+package jsonrpc
+
+import (
+	"bytes"
+	"encoding/json"
+)
+
+var (
+	// IndentOutput controls wether or not JSON output is indented.
+	IndentOutput = true
+	// EscapeHTML controls wether json output is HTML escaped.
+	EscapeHTML = false
+)
+
+// Marshal is a custom json marshaller that conditionally turns off html
+// escaping and applies indentation.
+func Marshal(v interface{}) ([]byte, error) {
```

Comment:
what does customMarshall distinguish this function from? the reason it is exposed is so that someone using the lib can make use of it. It's opinionated for sure, but that is why there are the vars above that will turn it back into just being a regular marshaller.